tests/multicomponent_compressible_projected_density_box.prm

```diff
@@ -0,0 +1,151 @@
+set Surface pressure                       = 5e10
+set Pressure normalization                 = surface
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Nonlinear solver tolerance             = 1e-12
+set Max nonlinear iterations               = 25
+set Output directory                       = visco_plastic_compressible_box
+
+# This calculates the adiabatic temperature profile down the model and is used
+# by the initial temperature and composition models.
+subsection Adiabatic conditions model
+  subsection Compute profile
+    set Number of points              = 100
+    set Composition reference profile = function
+  end
+end
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance = 1e-12
+  end
+end
+
+# The projected density approximation requires prescribed fields
+subsection Compositional fields
+  set Number of fields = 1
+  set Names of fields  = density_field
+  set Compositional field methods = prescribed field
+  set Types of fields             = density
+end
+
+# This is the functionality being tested. The projected density approximation
+# is the preferred compressibility formulation, as demonstrated by
+# the compressibility formulations benchmark (Gassmoeller et al.).
+# The composition statistics postprocessor requires that the single field we have
+# be filled, which ensures that the density we see has been prescribed.
+subsection Formulation
+  set Mass conservation    = projected density field
+  set Temperature equation = real density
+end
+
+subsection Initial composition model
+  set Model name = adiabatic density
+end
+
+# 1x1 m box with a 2x2 grid (which is the minimum)
+# there are no global or adaptive refinement steps
+# so this is the initial and end resolution.
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set Y extent = 1
+    set X extent = 1
+    set X repetitions = 1
+    set Y repetitions = 1
+  end
+end
+
+# We only want to evaluate the density due to our applied surface
+# pressure, so we disable gravity.
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 0.0
+  end
+end
+
+############### Boundary conditions
+# We fix the temperature on all boundaries, and as our model is so small this
+# prescribes the temperature throughout the model.
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = left, right, top, bottom
+  set List of model names = box
+
+  subsection Box
+    set Left temperature   = 1600
+    set Right temperature  = 1600
+    set Top temperature    = 1600
+    set Bottom temperature = 1600
+  end
+end
+
+# We allow free slip on all sides
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom,top, left, right
+end
+
+subsection Initial temperature model
+  set Model name = adiabatic
+end
+
+# These are the material model parameters used in composite
```

Comment:
I think it would be ok to remove these two lines (at minimum remove reference to visco plastic)